### DISCUSSION TITLE
Any way to fix this weird text rendering in hyprlock?

### DISCUSSION THREAD

**Discussion by chikenpotpi** [hyprwm/Hyprland#7407] [Q&A]
![image](https://github.com/user-attachments/assets/4936f1df-f70f-461e-b89d-a198eba4060e)

Compared to text in any applications, the edges look like an chromatic aberration effect was applied to it.
Happens in groupbar titles too.

**Comment by polluxcodes** [comment]
That's subpixel anti-aliasing. You need to disable it in fontconfig, which Hyprland uses indirectly through pango.

See `man fonts-config` for the location of the font configuration files on your computer, and add the following snippet to one of the `.conf` files.

```xml
<match target="font">
    <edit name="rgba" mode="assign">
        <const>none</const>
    </edit>
</match>
```
